src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs

```diff
@@ -971,6 +971,25 @@ public void ClickUnselectedCheckbox()
             }
         }
 
+        public void ClickSelectedCheckbox()
+        {
+            var checkedCheckboxes = Driver.FindElements(Objects.Pages.Common.Checkbox).Where(s => bool.Parse(s.GetProperty("checked"))).Count();
+            var checkboxes = Driver.FindElements(Objects.Pages.Common.Checkbox);
+            foreach (var checkbox in checkboxes)
+            {
+                if (bool.Parse(checkbox.GetProperty("checked")))
+                {
+                    checkbox.Click();
+
+                    if (!bool.Parse(checkbox.GetProperty("checked")))
+                    {
+                        Wait.Until(d => d.FindElements(Objects.Pages.Common.Checkbox).Where(s => bool.Parse(s.GetProperty("checked"))).Count() != checkedCheckboxes + 1);
+                        break;
```

Comment:
Not sure I understand what you're trying to do here. Can you elaborate?

Reply:
...checked. I don't really know what I am doing with that as I don't need it. 
 I have removed.

Reply:
All three implemented.

Reply:
checking through again 
Can you see anything else amiss?

Reply:
Not from my look through I didn't

Reply:
sent back